src/expander.rs

```diff
@@ -332,6 +354,25 @@ mod test {
             .unwrap()
     }
 
+    fn parcel_feature_value<'a>(
```

Comment:
Ooo... we should add functionality like this to the Bindle library. This would be generally useful.